data/templates/default.latex

```diff
@@ -328,9 +328,9 @@ $if(lang)$
 \ifxetex
   % Load polyglossia as late as possible: uses bidi with RTL langages (e.g. Hebrew, Arabic)
   \usepackage{polyglossia}
-  \setmainlanguage[$polyglossia-lang.options$]{$polyglossia-lang.name$}
+  \setmainlanguage[$for(polyglossia-lang.options)$$polyglossia-lang.options$$sep$,$endfor$]{$polyglossia-lang.name$}
 $for(polyglossia-otherlangs)$
```

Comment:
Note: a nicer syntax that now works is
```
$for(ployglossia-lang.options)$$it$$sep$,$endfor$
```

Reply:
Thanks for the hint!

Since part of the rationale for this was aligning it with the rest of the template in handling LaTeX package options: Should this also be changed in all other places in the template where a similar syntax is used? Or just in this instance?

Reply:
You're probably right that it's best to keep it uniform.
Maybe we should just merge it as it is, and perhaps think about updating the template more generally in another commit.